python_modules/dagster/dagster/_core/pipes/subprocess.py

```diff
@@ -93,17 +96,17 @@ def run(
         """
         with open_pipes_session(
             context=context,
-            context_injector=self.context_injector,
-            message_reader=self.message_reader,
+            context_injector=self._context_injector,
+            message_reader=self._message_reader,
```

Comment:
not a big deal, but why change these to the private accessors?

Reply:
i was renaming `env` to `base_env` and figured that these members probably are not intentionally part of the "public" api so moved them to private to be clear.